Automattic/wp-calypso
# Login: Add verify with code handling for 2FA

This is how it looks on production at the moment:

<img width="394" alt="screen_shot_2017-05-09_at_14 15 47" src="https://cloud.githubusercontent.com/assets/699132/25852969/710f0c58-34cb-11e7-875c-049670591712.png">

Calypso version which needs some more love:

<img width="475" alt="screen shot 2017-05-09 at 15 05 20" src="https://cloud.githubusercontent.com/assets/699132/25852924/453ccfa2-34cb-11e7-8ef6-9e37952d3211.png">

### Testing

1. Go to http://calypso.localhost:3000/login.
2. Use credentials for your account that has 2FA based on the Authenticator app enabled and WordPress.com app installed on the same device.
3. Once you get push notification click `Verify with code instead` button.
4. Enter code from the Authenticator app.
5. You should login.

### Review
